ui/src/utils/fetchWrappers/pipelineViewFetch.ts

```diff
@@ -0,0 +1,400 @@
+import { useCallback, useEffect, useMemo, useState } from "react";
```

Comment:
Can we rename the dir with hooks?

Reply:
fetcherHooks may be

Reply:
redefined fetcherHooks under Utils separately